src/tracer.js

```diff
@@ -206,16 +204,13 @@ export default class Tracer {
         let rpcServer = (spanKindValue === opentracing_tags.SPAN_KIND_RPC_SERVER);
 
 
-        // $FlowIgnore - I just want a span context up front.
         let ctx: SpanContext = new SpanContext();
         let samplerTags: any = {};
-        let debugRequest = (parent && parent.isDebugIDContainerOnly());
```

Comment:
nit: do you mind if we keep this variable even if we only use it once on line 212? it makes it slightly more readable